### ISSUE TITLE
Post create: Column 'slug' cannot be null

### ISSUE THREAD

**Issue by HenoTaMyJIb** [thedevdojo/voyager#1734]
- Laravel Version: 5.5.4
- Voyager Version: master (7f7c23a)
- PHP Version: 7.1.8
- Database Driver & Version: mysql

### Description:
Multilingual site is enabled. On post create I'm getting error `Integrity constraint violation: 1048 Column 'slug' cannot be null`.
In VoyagerBreadController store method, request object slug_i18n property is null for each language, I guess here is the problem. Before form submit, slug_i18n field is not updated?
```
[
  "_token" => "BdoALUBnaVSAdJbENXKoYC4DTxaoKpKwnlPUCQYa"
  "title_i18n" => "{"en":"My new post","et":"","ru":""}"
  "title" => "My new post"
  "body_i18n" => "{"en":"<p>Post content</p>"}"
  "body" => "<p>Post content</p>"
  "excerpt_i18n" => "{"en":null,"et":null,"ru":null}"
  "excerpt" => null
  "slug_i18n" => "{"en":null,"et":null,"ru":null}"
  "slug" => "my-new-post"
  "status" => "PUBLISHED"
  "category_id" => "1"
  "meta_description_i18n" => "{"en":null,"et":null,"ru":null}"
  "meta_description" => null
  "meta_keywords_i18n" => "{"en":null,"et":null,"ru":null}"
  "meta_keywords" => null
  "seo_title_i18n" => "{"en":null,"et":null,"ru":null}"
  "seo_title" => null
]
```
Same for post update, if slug is changed, changes are not saved to database.

### Steps To Reproduce:
Install voyager
Enable multilingual site
Add languages
Create new post

**Comment by lancepioch** [comment]
Labels: locale, possible bug

**Comment by philmmoore** [comment]
Yep, I'm seeing this too.

**Comment by marktopper** [comment]
This seems to have been fixed in #1802

**Comment by philmmoore** [comment]
Awesome 👍
